dbdreader/dbdreader.py

```diff
@@ -1553,7 +1555,13 @@ def __update_dbd_inventory(self, cacheDir):
         filenames=list(self.filenames)
         missing_cacheIDs = []
         for fn in self.filenames:
-            dbd=DBD(fn, cacheDir, raise_exception_if_cac_not_found=False)
+            try:
+                dbd=DBD(fn, cacheDir, raise_exception_if_cac_not_found=False)
+            except Exception as e:
+                logger.warning('File %s could not be loaded', fn)
+                logger.debug('Exception was %s', e)
+                filenames.remove(fn)
+                continue
```

Comment:
So this looks like

```
WARNING:dbdreader.py:File ./delayed_raw/02070000.EBD could not be loaded
DEBUG:dbdreader.py:Exception was ./delayed_raw/02070000.EBD seems not to be a valid DBD file
```
This file failed because it was empty...